src/model/EvaluationSummary.py

```diff
@@ -0,0 +1,35 @@
+from Utils.utils import (
+    get_human_verified_results,
+    count_actual_values,
+    count_predicted_values,
+    calculate_confusion_matrix_metrics, 
+    get_predicted_summary,
+    get_metrics
+)
+
+
+class EvaluationSummary:
+    """
+    Encapsulates the evaluation results, calculates confusion
+    matrix metrics and performance metrics based on summary_data.
+
+    Expects summary_data to be a list of (issue, summary_info) tuples.
+    """
+    def __init__(self, summary_data):
+        self.summary_data = summary_data
+        self.predicted_summary = get_predicted_summary(summary_data)
+        self.ground_truth = get_human_verified_results()
```

Comment:
It feels like this line should not belong inside a constructor. What if, after calculating all the results for 4hrs and then in this line we find out human results xlsx file is not available? 

So, I think we should verify that the human results are available even before we start the whole pipeline. wdyt?